File: app/src/main/java/com/novacenter/app/data/model/TurnoDTO.kt
package com.novacenter.app.data.model

import com.google.gson.annotations.SerializedName

data class TurnoDTO(
    val id: Int,
    val pacienteId: Int,
    val medicoId: Int,
    val fecha: String, // o LocalDateTime si usás adaptador
    val paciente: Persona?,
    val medico: Medico?
)

File: ApiCentroSalud/DTOs/TurnoDto.cs
namespace ApiCentroSalud.DTOs
{
    public class TurnoDto
    {
        public int Id { get; set; }
        public int PacienteId { get; set; }
        public int MedicoId { get; set; }
        public DateTime Fecha { get; set; }
        public PacienteDto Paciente { get; set; }
        public MedicoDto Medico { get; set; }
    }
}
